docs/_layouts/homepage.html

```diff
@@ -48,6 +48,7 @@ <h1>Highlights</h1>
                 <div class="description">
                     <p>This suite of packages provides simple and intuitive classes to parse, validate, format and manipulate URIs and their components. It is built to enable working with
```

Comment:
I would remove "simple" or "easy" whenever I see them. In retrospective I would not have added them if I knew then what I know now 😄 . Intuitive can stay

Reply:
>  In retrospective I would not have added them if I knew then what I know now :smile:

Oh, I've walked that road before too. :smile: